dbms/src/TestUtils/mockExecutor.h

```diff
@@ -101,6 +103,9 @@ class DAGRequestBuilder
     DAGRequestBuilder & sort(MockOrderByItem order_by, bool is_partial_sort, uint64_t fine_grained_shuffle_stream_count = 0);
     DAGRequestBuilder & sort(MockOrderByItemVec order_by_vec, bool is_partial_sort, uint64_t fine_grained_shuffle_stream_count = 0);
 
+    void setCollation(Int32 collator_) { properties.collator = -abs(collator_); }
```

Comment:
may be we can wrap a function for `-abs(collator_)`
```
/// add some comments
Int32 inline getXXXCollation(Int32 collator_)
{
    return -abs(collator_);
}
```

Reply:
> may be we can wrap a function for `-abs(collator_)`
> 
> ```
> /// add some comments
> Int32 inline getXXXCollation(Int32 collator_)
> {
>     return -abs(collator_);
> }
> ```

ok